src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -310,6 +356,160 @@ internal override void SetDefaultSourcePath(string defaultPath)
             WriteValueToFile<string>(fileName, "DefaultSourcePath", defaultPath);
         }
 
+#if UNIX
+        /// <summary>
+        /// Gets the identity name to use for writing to syslog.
+        /// </summary>
+        /// <returns>
+        /// The string identity to use for writing to syslog.
+        /// <para>
+        /// The default value is 'powershell'
+        /// </para>
+        /// </returns>
+        internal override string GetSysLogIdentity()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            string identity = ReadValueFromFile<string>(fileName, "LogIdentity");
+            if
+            (
+                string.IsNullOrEmpty(identity)
+                ||
+                StringComparer.OrdinalIgnoreCase.Compare(LogDefaultValueName, identity) == 0
+            )
+            {
+                identity = "powershell";
+            }
+            return identity;
+        }
+
+        /// <summary>
+        /// Gets the log level filter.
+        /// </summary>
+        /// <returns>One of the PSLevel values indicating the level to log.
+        /// <para>
+        /// The default value is PSLevel.Informational
+        /// </para>
+        /// </returns>
+        internal override PSLevel GetLogLevel()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            string levelName = ReadValueFromFile<string>(fileName, "LogLevel");
+            PSLevel level;
+            if
+            (
+                StringComparer.OrdinalIgnoreCase.Compare(LogDefaultValueName, levelName) == 0
+                ||
+                !Enum.TryParse<PSLevel>(levelName, true, out level )
+            )
+            {
+                level = PSLevel.Informational;
+            }
+            return level;
+        }
+
+        /// <summary>
+        /// The supported separator characters for listing channels and keywords in configuration.
+        /// </summary>
+        static readonly char[] s_valueSeparators = new char[] {' ', ',', '|'};
+
+        /// <summary>
+        /// Provides a string name to indicate the default for a configuration setting.
+        /// </summary>
+        const string LogDefaultValueName = "default";
+
+        const PSChannel DefaultChannels = PSChannel.Operational;
+
+        /// <summary>
+        /// Gets the bitmask of the PSChannel values to log.
+        /// </summary>
+        /// <returns>
+        /// A bitmask of PSChannel.Operational and/or PSChannel.Analytic
```

Comment:
Please add final dot.

Reply:
In the future, it would help if you provided a single feedback indicating DOC comments are missing periods at the end of sentences instead of pointing out each one individually.  It makes the feedback somewhat tedious to weed through.  Thanks.

Reply:
I agree. I was surprised that sometimes I saw a dot in the XML commentary, so I thought it was just a few typos.